src/librustc/middle/mem_categorization.rs

```diff
@@ -184,7 +184,7 @@ pub enum Note {
 // dereference (`@T`). So use `cmt.ty` to find the type of the value in
 // a consistent fashion. For more details, see the method `cat_pattern`
 #[derive(Clone, Debug, PartialEq)]
-pub struct cmt_<'tcx> {
```

Comment:
The doc comment `` `cmt`: "Category, Mutability, and Type".`` need to be changed.

Reply:
Also change other comments contain `` `cmt` ``.

Reply:
This is updated in the later commits